src/frontend/src/pages/AdminToolsPage/AdminToolsUserManagement.tsx

```diff
@@ -12,6 +14,7 @@ import LoadingIndicator from '../../components/LoadingIndicator';
 import ErrorPage from '../ErrorPage';
 import { fullNamePipe } from '../../utils/pipes';
 import { RoleEnum, User } from 'shared';
+import NERAutocomplete from '../../components/NERAutocomplete';
 
 const AdminToolsUserMangaement: React.FC = () => {
```

Comment:
a weird thing with the role management rn is when you change someone's role, it leaves their old role in the search bar since it's part of what's displayed now. I would say either we don't display the role in the search bar, clear the search bar when you change a role, or update the role displayed in the search bar when it's changed

Reply:
it is very important to james that we show the role in the search bar so i'll make it deselect once you update